test/start-stop.make

```diff
@@ -1,68 +1,23 @@
 # Brings up the emulator environment:
-# - creates pmem NVDIMM files if necessary
-# - starts QEMU virtual machines with pmem NVDIMMs as described in https://github.com/qemu/qemu/blob/bd54b11062c4baa7d2e4efadcf71b8cfd55311fd/docs/nvdimm.txt
-# - starts a Kubernetes cluster
+# - starts a Kubernetes cluster with NVDIMMs as described in https://github.com/qemu/qemu/blob/bd54b11062c4baa7d2e4efadcf71b8cfd55311fd/docs/nvdimm.txt
 # - generate pmem secrets if necessary
-# - installs pmem-csi driver
-#
-# It operates on the cluster in _work/$(CLUSTER).
-start: _work/$(CLUSTER)/created _work/$(CLUSTER)/start-kubernetes _work/$(CLUSTER)/run-qemu _work/$(CLUSTER)/ssh test/setup-ca-kubernetes.sh _work/.setupcfssl-stamp
+start: test/setup-ca-kubernetes.sh _work/.setupcfssl-stamp
```

Comment:
Previously it was possible to do `make stop && make push-test-image && make start` to reboot the machine with a new image, without having to go through the expensive cluster setup process.

This command combination still works, but now re-creates the cluster. That's okay because it provides a simple way to reset the cluster. We just need something else for the developer use case above.

Can you add a `make restart`? It first needs to reboot each VM and then wait for them to come up again.

Rebooting the VMs can be done with `ssh sudo systemctl reboot`. Waiting can be done by checking with ssh and kubectl.

Reply:
I added `make restart` to reboot the cluster,  it waits for all the nodes to have ssh, then it verifies that all the kubernetes nodes are ready